### DISCUSSION TITLE
Add a note somewhere in the README that selinux enablement can lead to pods trying to use volumes with many files never booting

### DISCUSSION THREAD

**Discussion by sharkymcdongles** [mysticaltech/terraform-hcloud-kube-hetzner#1359] [Ideas]
### Description

I got bit by this, and it took awhile to figure out because you get very little in the way of logs about this outside of the containerd logs in the k3s folder on the node. These logs don't even make it to journalctl sadly adding further difficulty when investigating.

Long story short disabling selinux for a nodepool to use these volumes is the best workaround in the short term. Best solution will be this: https://kubernetes.io/blog/2023/04/18/kubernetes-1-27-efficient-selinux-relabeling-beta/

But for this we need this to be closed first:

https://github.com/hetznercloud/csi-driver/issues/582

**Comment by mysticaltech** [comment]
@sharkymcdongles PR welcome if you want to make our docs more complete with this special use case.

**Comment by mysticaltech** [comment]
Done, and also added a system to configure rules. See docs.
